packages/core-mobile/diagrams/Notifications.drawio

```diff
@@ -0,0 +1 @@
pre&gt;&lt;/div&gt;" style="rounded=1;whiteSpace=wrap;html=1;" vertex="1" parent="13"><mxGeometry x="210" y="156" width="120" height="60" as="geometry"/></mxCell><mxCell id="27" value="&lt;div style=&quot;background-color:#1e1f22;color:#bcbec4&quot;&gt;&lt;pre style=&quot;font-family:'JetBrains Mono',monospace;font-size:11.3pt;&quot;&gt;&lt;div&gt;&lt;pre style=&quot;font-family:'JetBrains Mono',monospace;font-size:11.3pt;&quot;&gt;&lt;span style=&quot;color:#a9b7c6;&quot;&gt;handleNotificationCallback&lt;/span&gt;&lt;/pre&gt;&lt;/div&gt;&lt;/pre&gt;&lt;/div&gt;" style="rounded=1;whiteSpace=wrap;html=1;" vertex="1" parent="13"><mxGeometry x="128" y="55" width="264" height="60" as="geometry"/></mxCell><mxCell id="16" style="edgeStyle=none;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;" edge="1" parent="1" source="14" target="22"><mxGeometry relative="1" as="geometry"><mxPoint x="-164" y="187.43957503320075" as="targetPoint"/></mxGeometry></mxCell><mxCell id="24" style="edgeStyle=none;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;dashed=1;" edge="1" parent="1" source="23" target="19"><mxGeometry relative="1" as="geometry"/></mxCell><mxCell id="65" value="observe" style="edgeLabel;html=1;align=center;verticalAlign=middle;resizable=0;points=[];" vertex="1" connectable="0" parent="24"><mxGeometry x="0.0597" y="3" relative="1" as="geometry"><mxPoint as="offset"/></mxGeometry></mxCell><mxCell id="30" style="edgeStyle=none;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;" edge="1" parent="1" source="28" target="27"><mxGeometry relative="1" as="geometry"/></mxCell><mxCell id="32" style="edgeStyle=none;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;" edge="1" parent="1" source="31" target="19"><mxGeometry relative="1" as="geometry"/></mxCell><mxCell id="33" value="TAPS" style="edgeLabel;html=1;align=center;verticalAlign=middle;resizable=0;points=[];" vertex="1" connectable="0" parent="32"><mxGeometry x="-0.1354" y="2" relative="1" as="geometry"><mxPoint as="offset"/></mxGeometry></mxCell><mxCell id="31" value="User" style="shape=umlActor;verticalLabelPosition=bottom;verticalAlign=top;html=1;outlineConnect=0;" vertex="1" parent="1"><mxGeometry x="78.5" y="733" width="30" height="60" as="geometry"/></mxCell><mxCell id="66" style="edgeStyle=none;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=0;exitY=0.25;exitDx=0;exitDy=0;entryX=0.25;entryY=1;entryDx=0;entryDy=0;" edge="1" parent="1" source="19" target="23"><mxGeometry relative="1" as="geometry"><Array as="points"/></mxGeometry></mxCell><mxCell id="68" value="&lt;pre style=&quot;color: rgb(188, 190, 196); font-family: &amp;quot;JetBrains Mono&amp;quot;, monospace; font-size: 11.3pt;&quot;&gt;type, detail&lt;/pre&gt;" style="edgeLabel;html=1;align=center;verticalAlign=middle;resizable=0;points=[];" vertex="1" connectable="0" parent="66"><mxGeometry x="-0.2233" y="-3" relative="1" as="geometry"><mxPoint x="-26" y="-21" as="offset"/></mxGeometry></mxCell></root></mxGraphModel></diagram></mxfile>
```

Comment:
can you add a screenshot of this nice diagram too?

Reply:
Will be updated in following PR